onelogin/resource_onelogin_app_rules.go

```diff
@@ -17,14 +19,14 @@ import (
 // AppRules returns a resource with the CRUD methods and Terraform Schema defined
 func AppRules() *schema.Resource {
 	return &schema.Resource{
-		Create: appRuleCreate,
-		Read:   appRuleRead,
-		Update: appRuleUpdate,
-		Delete: appRuleDelete,
+		CreateContext: appRuleCreate,
```

Comment:
A lot of the additional changes in this file are something we can do as we need to make changes into the different resources we define. We might want to make an effort to update the rest all at once at some point, as it is needed for logging to work properly.